### ISSUE TITLE
TypeError: answer.toLowercase is not a function

### ISSUE THREAD

**Issue by rplankenhorn** [pmadruga/react-native-clean-project#62]
Version: >3.6.1

```
npx react-native-clean-project
npx: installed 1 in 1.551s
Wipe iOS build folder? (Y/n) y
readline.js:1154
            throw err;
            ^

TypeError: answer.toLowercase is not a function
    at checkAnswer (/Users/<User>/.npm/_npx/16899/lib/node_modules/react-native-clean-project/source/internals/options.js:57:19)
    at /Users/<User>/.npm/_npx/16899/lib/node_modules/react-native-clean-project/source/internals/options.js:77:22
    at /Users/<User>/.npm/_npx/16899/lib/node_modules/react-native-clean-project/source/internals/options.js:52:5
    at Interface._onLine (readline.js:335:5)
    at Interface._line (readline.js:666:8)
    at Interface._ttyWrite (readline.js:1010:14)
    at ReadStream.onkeypress (readline.js:213:10)
    at ReadStream.emit (events.js:315:20)
    at emitKeys (internal/readline/utils.js:345:14)
    at emitKeys.next (<anonymous>)
```

I downgraded to 3.6.1 and it works so this is clearly an issue with the latest release.

**Comment by 4RGUS** [comment]
Will you let me know how you downgraded it, as I don't see it in my package.json file

**Comment by 4RGUS** [comment]
> Will you let me know how you downgraded it, as I don't see it in my package.json file

No worries, I just did "yarn add react-native-clean-project@3.6.1"

**Comment by pmadruga** [comment]
Sorry guys. I let a PR went in yesterday without proper testing. Should be fixed now in version 3.6.4
